mitremap-notebook/requirements.txt

```diff
@@ -0,0 +1,146 @@
+adal==1.2.7
```

Comment:
Do you really need all of these???
A lot seem to be driven by msticpy requirements. 
It's probably fine if people are creating their own v environment but could cause a bit of chaos in AML.
I'm a bit torn on this question really.
You could provide a requirements.txt that just contains the following:
```
ipywidgets==7.5.1
transformers==4.5.1
torch==1.10.2
msticpy==2.1.2
nltk==3.6.2
iocextract==1.13.1
shap==0.41.0
```
And another requirements-stable.txt or requirements-full.txt.
Add a note in the readme file that the notebook and code has been tested with the requirements-full.txt dependencies. If installing into a pre-built environment such as Azure Machine Learning you should try first with the requirements.txt. 
If this doesn't work, revert to the requirements-full.txt. Installing this may cause more conflicts with other packages installed in the environment so be prepared to dedicate the AML Compute to this notebook code.

Reply:
Resolved.